src/elements/subscribe-dialog.html

```diff
@@ -144,12 +163,19 @@
                 '_handleDialogToggled(opened, data)',
                 '_handleSubscribed(subscribed)'
             ];
-        }  
+        } 
+
+        ready() {
+            super.ready();
+            this.initialHeight = window.innerHeight; 
+        } 
 
         constructor() {
-            super();                     
+            super();  
             this.addEventListener('iron-overlay-canceled', this._close);
-        }
+            this.addEventListener('iron-resize', this._resize);
+            window.addEventListener('resize', this._windowResize.bind(this));
```

Comment:
add/remove listeners on `connectedCallback()` and  `disconnectedCallback()`  respectively
```
connectedCallback() {
        super.connectedCallback();
        window.addEventListener('reset-query-params', this._removeQueryParams);
      }
      disconnectedCallback() {
        super.disconnectedCallback();
        window.removeEventListener('reset-query-params', this._removeQueryParams);
      }
```